scrapy/core/engine.py

```diff
@@ -15,7 +15,11 @@
 
 from scrapy import signals
 from scrapy.core.scraper import Scraper
-from scrapy.exceptions import DontCloseSpider, ScrapyDeprecationWarning
+from scrapy.exceptions import (
+    CloseSpider,
+    DontCloseSpider, 
+    ScrapyDeprecationWarning,
+)   
```

Comment:
```suggestion
    DontCloseSpider,
    ScrapyDeprecationWarning,
)
```
Remove the trailing whitespaces that are making the flake8 check fail.